storage/scylla_api.h

```diff
@@ -0,0 +1,199 @@
+/*
+ * Copyright (C) 2019 BiiLabs Co., Ltd. and Contributors
+ * All Rights Reserved.
+ * This is free software; you can redistribute it and/or modify it under the
+ * terms of the MIT license. A copy of the license can be found in the file
+ * "LICENSE" at the root of this distribution.
+ */
+#ifndef TA_SCYLLA_API_H_
+#define TA_SCYLLA_API_H_
+#ifdef __cplusplus
+extern "C" {
+#endif
+
+#include "accelerator/common_core.h"
+#include "cassandra.h"
+#include "scylla_config.h"
+
+#define BUNDLE_1                                                               \
+  "BUNDLE100000LIKCEJTTIQOTTAWSQCCQQ9A9VOKIWRBWVPXMCGUENWVVMQAMPEIVHEQ9JXLCNZ" \
```

Comment:
I think the following MACROs are defined for testing. If so, it may be better to move them into directory `/tests`